ui/src/components/Auth/LoggedIn.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019 The FoodUnit Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Link} from "@reach/router";
+import {LOGIN_ROUTE} from "../../util/Routes";
+import {inject, observer} from "mobx-react";
+import React from 'react';
+
+const LoggedIn = inject('auth')(observer(({auth, children}) => {
```

Comment:
What does `inject('auth')(observer(({auth, children})` do? :D

Reply:
it injects the aut model and makse the component a mobx observer. {auth, children} extracts auth and children from the props object

Reply:
Achja, Object Destruction oder wie das hieß.